turkey/templates/_base.html

```diff
@@ -35,7 +35,7 @@
 <!-- Register on login page -->
 <!-- Forgot username/password on login page -->
 <!-- Allow email login on login page -->
-          <p class="navbar-text">Hi, you can <a href="{{ url_for('register') }}">register</a> or </p>
+          <p class="navbar-text">Hi, you can <a id="register-link" href="{{ url_for('register') }}">register</a> or </p>
```

Comment:
The merge conflict is on this line- I was building in registration blocking capabilities while I was travelling (sorry!).